src/_C005.ino

```diff
@@ -63,6 +63,15 @@ bool CPlugin_005(byte function, struct EventStruct *event, String& string)
           if (lastPartTopic == F("cmd")) {
             cmd = event->String2;
             parseCommandString(&TempEvent, cmd);
+
+String log = F("SP_C005a: string=");
+log += string;
+log += F(" ;cmd=");
+log += cmd;
+log += F(" ;taskIndex=");
+log += event->TaskIndex;
+addLog(LOG_LEVEL_INFO, log);
```

Comment:
Please use proper indentation and wrap in `if(logLevelActiveFor)...`

Reply:
work-in-progress